backend/docker_entrypoint.sh

```diff
@@ -8,15 +8,20 @@ BE_PORT=${BACKEND_PORT:-8000}
 
 poetry config virtualenvs.path /backend/virtualenvs
 
+poetry lock --no-update
```

Comment:
It isn't mandatory, but I think it improves the dx, since whenever we update `backend/pyproject.toml` now it'll install the new packages when restarting the container, without any additional commands.